content/post/hugo_ga_preview.adoc

```diff
@@ -0,0 +1,218 @@
+---
+draft: true
+title: 'Using GitHub Actions to build automagic Hugo previews of draft articles'
+date: "2022-04-05T20:35:13Z"
+image: "/images/2022/04/build.png"
+thumbnail: "/images/2022/04/IMG_5139.jpeg"
+credit: "https://twitter.com/rmoff/"
+categories:
+- Hugo
+- GitHub
+---
+
+:source-highlighter: rouge
+:icons: font
+:rouge-css: style
+:rouge-style: github
+
+This blog is written in Asciidoc, built using Hugo, and hosted on GitHub Pages. I recently wanted to share the draft of a post I was writing with someone and ended up exporting a local preview to a PDF - not a great workflow! This blog post shows you how to create an automagic hosted preview of any draft content on Hugo using GitHub Actions.
+
+<!--more-->
+
+I link:/2020/12/20/scheduling-hugo-builds-on-github-pages-with-github-actions/[wrote previously] about using GitHub Actions to automagically build and deploy my blog whenever I push new content to the GitHub repo that hosts the source. My friend https://twitter.com/gunnarmorling[Gunnar Morling] does not only that with https://www.morling.dev/[his blog], but also a neat preview functionality for new content in a Pull Request (PR). I'm writing this blog to document the steps I took to mimic https://github.com/gunnarmorling/morling.dev/blob/master/.github/workflows/preview.yml[his setup] :) 
+
+In essence it's the same as link:/2020/12/20/scheduling-hugo-builds-on-github-pages-with-github-actions/[deploying the production Hugo site to GitHub Pages], except it's hosted on https://surge.sh/[`https://surge.sh/`] instead and with a variable URL (since you might have multiple drafts each with their own PR). 
+
+== Setting up an account on surge.sh
+
+https://surge.sh/[`https://surge.sh/`] is not a shell script as the name may suggest, but a hosted service that provides `Static web publishing for Front-End Developers`. Their audience is denoted very clearly by the fact you can't sign up online - they assume that _of course_ you'll have `npm` installed (because you're a front-end developer, right). For us heathens without it, you need to install `npm` first which you can do via `brew` (which in turn you can install using `npm`…only kidding)
```

Comment:
`Static web publishing for Front-End Developers` -- Is that formatted as code intentionally? Or should it rather be regular quotes? Same for the formatting of https://surge.sh, but then I'm not sure how you go about formatting such things.